Show number of events a competitor is registered for - fixes #430

pingskills opened the pull request:
![registrations](https://cloud.githubusercontent.com/assets/4403168/15392387/218e082e-1e0a-11e6-8a37-bc9799264e45.png)

pedrosino commented on WcaOnRails/app/views/registrations/index.html.erb:
```diff
@@ -19,6 +19,8 @@
         <% end %>
 
+        <!-- Extra column for displaying number of events registered for -->
+        <th></th>
```
Maybe write something in this header, like "Total" or "Total events" or "Events"

jonatanklosko commented on WcaOnRails/app/views/registrations/index.html.erb:
```diff
@@ -48,6 +50,9 @@
           <% end %>
 
+          <td>
+            <%= registration.events.length %>
```
Since we have the separate table for registration events the most optimal way to compute the registration events count is `registration.registration_events.count`. This will grab the count directly from database, whereas what you've done will grab the whole events from database then map them with Event objects and finally get length of the array.

pingskills replied:
Hi @jonatanklosko, does it make a difference if the view has already fetched the whole events from the database and mapped them with the Event object? The view already does `<% if registration.events.include?(event) %>`

Or is it still better to use the `registration.registration_events.count`?

jonatanklosko replied:
Yeah, I think it's still better. Note that even if we do eager loading so that we don't perform another query for the events, the `events` method perform the mapping anyway.

pingskills commented:
![registration-count](https://cloud.githubusercontent.com/assets/4403168/15397927/2a5be64a-1e25-11e6-8db2-4fed0e804de3.png)

I've updated the way the counts are displayed thanks to @jonatanklosko and I've added a heading as suggested by @pedrosino 

I added some css to right justify the column and made it sortable. Do you think this is a good idea?

jonatanklosko commented:
LGTM!
